src/components/sbb-accordion/sbb-accordion.scss

```diff
@@ -3,54 +3,3 @@
 // Default component properties, defined for :host. Properties which can not
```

Comment:
Since the expansion-panel can be shown alone but also inserted into the accordion, I think that it is better to have its style "self defined", keeping all its rules within the expansion-panel itself, rather than have this style divided into different files (and different origins)

Anyway I tried different solutions, but since a compound selector doesn't work with :host, :slotted, etc, the best solution in my opinion is to change `:host(:first-child)` with `:host(:first-of-type)`, tried that and it works fine